js/Tabs/Scroller.js

```diff
@@ -0,0 +1,87 @@
+class Scroller {
```

Comment:
This is kind of a helper that kinda runs something like an animation in order to handle scrolling of tabs when using mousewheel. Built in-house. Because we can